docs/stylesheets/extra.css

```diff
@@ -0,0 +1,3 @@
+:root {
+  --md-primary-fg-color:        rgb(18, 97, 141);
```

Comment:
You didn't like any of the default colors? https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#primary-color

Reply:
used the color from Arturo web site 🤷‍♂️